Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -124,12 +126,12 @@ def wait_for_uninstallation_to_complete(client: demisto_client):
     retry = 0
     sleep_duration = 150
     try:
-        installed_packs = get_all_installed_packs(client)
+        installed_packs = get_all_installed_packs(client, unremovable_packs)
         # Monitoring when uninstall packs don't work
         installed_packs_amount_history, failed_uninstall_attempt_count = len(installed_packs), 0
         # new calculation for num of retries
         retries = math.ceil(len(installed_packs) / 2)
-        while len(installed_packs) > len(UNREMOVABLE_PACKS):
+        while len(installed_packs) > len(unremovable_packs):
```

Comment:
i think set operation here is preferred:
```
while len(installed_packs - unremoveable_packs)
```
As we check explicitly that whatever is left is only the unremovable.